### PR TITLE
Refactor to use super._sendDoneEvent

### PR DESCRIPTION
**Pull request by andrecardoso**

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(2) Add common PUD code to RiseElement and refactor components](https://trello.com/c/xCYXye6Y/153-2-add-common-pud-code-to-riseelement-and-refactor-components)

**Comment by andrecardoso** [comment]
[(1) Create PUD documentation for designers](https://trello.com/c/rNofaTMw/173-1-create-pud-documentation-for-designers)

File: README.md [hunk 1]
```diff
@@ -21,6 +21,20 @@ The component sends the following events:
 
 - **_configured_**: The component has initialized what it requires to and is ready to handle a _start_ event.
 
+### Usage
+
+Include the tag in the document and call `reportDone()` when necessary (on animation end, on data error, etc):
```
Comment (olegrise) [inline README.md]: @andrecardoso I would say "Include the **`<rise-play-until-done>`** tag ..."

**Comment by andrecardoso** [comment]
@olegrise Fixed. Please re-review. Thanks.

**Pull request merged**